packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/codec/codec.controller.js

```diff
@@ -186,15 +185,21 @@ export default /* @ngInject */ function TelecomTelephonyLinePhoneCodecCtrl(
       services,
       (service) => ['sip', 'mgcp'].indexOf(service.featureType) > -1,
     );
-
-    return tucVoipLinePhone.fetchAll().then((voipLinePhones) =>
-      filter(filteredServices, (service) =>
-        some(voipLinePhones, {
-          serviceName: service.serviceName,
-          billingAccount: service.billingAccount,
+    return $q
+      .all(
+        filteredServices.map((service) => {
+          return OvhApiTelephony.Line()
+            .Phone()
+            .v6()
+            .get({
+              billingAccount: service.billingAccount,
+              serviceName: service.serviceName,
+            })
+            .$promise.then(() => service)
+            .catch(() => null);
         }),
-      ),
-    );
+      )
+      .then((results) => compact(results));
```

Comment:
Is compact necessary? If you want to remove null values u could just do : 

```
.then((results) => results.filter((result) => result))
```

It's better than importing another thing from lodash again